smrf/framework/model_framework.py

```diff
@@ -246,11 +238,35 @@ def __init__(self, config, external_logger=None):
             self._logger.info(getqotw())
 
         # Initialize the distribute dict
-        self._logger.info('Started SMRF --> %s' % datetime.now())
+        self._logger.info('Started SMRF --> %s' % now)
         self._logger.info('Model start --> %s' % self.start_date)
         self._logger.info('Model end --> %s' % self.end_date)
         self._logger.info('Number of time steps --> %i' % self.time_steps)
 
+    def _setup_date_and_time(self):
+        self.time_zone = pytz.timezone(self.config['time']['time_zone'])
+        is_utz = self.time_zone == pytz.UTC
+
+        # Get the time section utils
+        self.start_date = pd.to_datetime(
+            self.config['time']['start_date'], utc=is_utz
+        )
+        self.end_date = pd.to_datetime(
+            self.config['time']['end_date'], utc=is_utz
+        )
+
+        if not is_utz:
+            self.start_date = self.start_date.tz_localize(self.time_zone)
+            self.end_date = self.end_date.tz_localize(self.time_zone)
+
+        # Get the time steps correctly in the time zone
+        self.date_time = pd.date_range(
+            self.start_date,
+            self.end_date,
+            freq=f"{self.config['time']['time_step']}min",
```

Comment:
I'm unfamiliar with this, and am getting a syntax error here. Is freq=f"{60}min" what was intended?

Reply:
What version of Python do you have?

Reply:
To give some background - it is called an `f-string` and was introduced in Python 3.6.
https://www.python.org/dev/peps/pep-0498/#id37

Reply:
Makes sense, I've got 3.5.